src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1302,7 +1336,11 @@ static std::vector<TestCollection> testSuite = {
   { create_init_test("hostmem-bw", "Run 'bandwidth kernel' when host memory is enabled", "bandwidth.xclbin"), hostMemBandwidthKernelTest },
   { create_init_test("bist", "Run BIST test", "verify.xclbin", true), bistTest },
   { create_init_test("vcu", "Run decoder test", "transcode.xclbin"), vcuKernelTest },
-  { create_init_test("aie-pl", "Run AIE PL test", "vck5000_pcie_pl_controller.xclbin.xclbin"), aiePlTest }
+  { create_init_test("aie", "Run PL controlled AIE test", "vck5000_pcie_pl_controller.xclbin.xclbin"), aiePlTest },
+  { create_init_test("ps-aie", "Run PS controlled AIE test", "ps_aie.xclbin"), psAieTest },
```

Comment:
Being that the xclbin names can be removed from this file (https://github.com/Xilinx/XRT/pull/6671#issuecomment-1182470368) can the xclbin entries be removed for the new PS kernel validation tests.  For example:
```suggestion
  { create_init_test("ps-aie", "Run PS controlled AIE test", ""), psAieTest },
```

Reply:
I removed the xclbin names for the tests where it is currently possible. Some (iops for example) will need later work to the hosts to remove the name from the xbutil source code.